Vector.c

```diff
@@ -79,10 +79,9 @@ bool Vector_countEquals(const Vector* this, unsigned int expectedCount) {
    return n == expectedCount;
 }
 
-Object* Vector_get(const Vector* this, int idx) {
-   assert(idx >= 0);
-   assert(idx < this->arraySize);
-   assert(idx < this->items);
+Object* Vector_get(const Vector* this, size_t idx) {
+   assert(idx < (size_t)this->arraySize);
+   assert(idx < (size_t)this->items);
```

Comment:
There should be a follow-up making the whole `Vector` implementation internally use `size_t`.

Reply:
Yep, I tried to limit the scope of the change here so we can (continue to) tackle this in small sections.